src/server.c

```diff
@@ -5039,7 +5039,7 @@ sds genRedisInfoString(const char *section) {
             server.stat_expired_time_cap_reached_count,
             server.stat_expire_cycle_time_used/1000,
             server.stat_evictedkeys,
-            server.stat_eviction_exceeded_time + current_eviction_exceeded_time,
+            (unsigned long long) server.stat_total_eviction_exceeded_time + current_eviction_exceeded_time,
```

Comment:
i think maybe we better convert both info fields to `ms`, or even `seconds` (not very useful in microseconds)